style.css

```diff
@@ -0,0 +1,311 @@
+  border-radius: 4px 0 0 4px;
+}
+
+.nav-text {
+  display: table-cell;
+  width: 100%;
+  overflow: hidden;
+  color: #dcdcdd;
+  vertical-align: middle;
+}
+
+.text {
+  display: block;
+  width: 100%;
+  height: 30px;
+  margin: 8px auto;
+  text-align: center;
+  color: #fff;
+  font-weight: 700;
+}
+
+.apple {
+  display: block;
+  width: 100%;
+  height: 30px;
+  margin: 3px auto;
+  text-align: center;
+  font-weight: 700;
+}
+
+.searchbox-wrapper {
+  padding: 8px 13px;
+  width: 13.6%;
+}
+
+.searchbox-container {
+  background-image: url(data:image/svg+xml;base64,PHN2ZyB2ZXJzaW9uPSIxLjEiIGJhc2VQcm9maWxlPSJmdWxsIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHhtbG5zOnhsaW5rPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5L3hsaW5rIiB4bWxuczpldj0iaHR0cDovL3d3dy53My5vcmcvMjAwMS94bWwtZXZlbnRzIj4KPGcgZmlsbC1vcGFjaXR5PSIuNSI%2BCgk8cG9seWdvbiBwb2ludHM9IjkuMjA3LDcuMTI2IDcuNzkzLDguNTQxIDExLjc5MywxMi41NDEgMTMuMjA3LDExLjEyNiIgLz4KCTxwYXRoIGQ9Ik01LjkxNywzYzEuNjA4LDAsMi45MTcsMS4zMDgsMi45MTcsMi45MTdTNy41MjUsOC44MzMsNS45MTcsOC44MzNTMyw3LjUyNSwzLDUuOTE3UzQuMzA4LDMsNS45MTcsMyBNNS45MTcsMQoJCUMzLjIwMSwxLDEsMy4yMDEsMSw1LjkxN3MyLjIwMSw0LjkxNyw0LjkxNyw0LjkxN3M0LjkxNy0yLjIwMSw0LjkxNy00LjkxN0MxMC44MzMsMy4yMDEsOC42MzIsMSw1LjkxNywxTDUuOTE3LDF6IiAvPgo8L2c%2BCjxnPgoJPHBvbHlnb24gZmlsbD0iI2ZmZiIgcG9pbnRzPSI5LjIwNyw2LjEyNiA3Ljc5Myw3LjU0MSAxMS43OTMsMTEuNTQxIDEzLjIwNywxMC4xMjYiIC8%2BCgk8cGF0aCBmaWxsPSIjZmZmIiBkPSJNNS45MTcsMmMxLjYwOCwwLDIuOTE3LDEuMzA4LDIuOTE3LDIuOTE3UzcuNTI1LDcuODMzLDUuOTE3LDcuODMzUzMsNi41MjUsMyw0LjkxN1M0LjMwOCwyLDUuOTE3LDIKCQkgTTUuOTE3LDBDMy4yMDEsMCwxLDIuMjAxLDEsNC45MTdzMi4yMDEsNC45MTcsNC45MTcsNC45MTdzNC45MTctMi4yMDEsNC45MTctNC45MTdDMTAuODMzLDIuMjAxLDguNjMyLDAsNS45MTcsMEw1LjkxNywweiIgLz4KPC9nPgo8L3N2Zz4K);
+  background-position: 6px 5px;
+}
+
+#searchbox {
+  width: 100%;
+  background-color: transparent;
+  border-radius: 10px;
+}
+
+.content-wrapper {
+  max-width: 1440px;
+  min-width: 800px;
+  margin: 0 auto;
+}
+
+.form-container {
+  border-radius: 10px;
+}
+
+.grid-wrapper {
+  max-width: 1440px;
+  min-width: 800px;
+  margin: 0 auto;
+}
+
+.hero {
+  background: url(https://web.archive.org/web/20140228114747im_/http://images.apple.com/v/home/as/images/your_verse_hero.jpg);
+  background-position: 50% 100%;
+  background-repeat: no-repeat;
+  background-size: 1440px 1315px;
+  text-align: center;
+  height: 70vh;
+  display: grid;
+  align-content: space-evenly;
+  object-fit: cover;
+}
+
+.grid-header {
+  position: relative;
+  min-width: 800px;
+}
+
+.grid-footer {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  grid-template-areas: "info1 info2 info3 info4";
+  grid-gap: 2px;
+  justify-content: center;
+  height: 200px;
+  max-width: 1440px;
+  margin: 0 auto;
+  overflow: hidden;
+}
+
+.hero-header {
+  padding: 50px 0;
+}
+
+.background-image {
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+}
+
+.main-header {
+  margin-bottom: -9px;
+}
+
+.head-img1 {
+  display: block;
+  height: 104px;
```

Comment:
- [ ] **Optional** This is additional if you want to do it: If you take out this height, the proportion of your image will be respected, so it doesn't look stretched